### PR TITLE
fix: postgresql CVE fix

### PR DESCRIPTION
**Pull request by SandhyaRavi2403**
**What problem does this PR solve?**:
Patch postgresql image with fixed CVE's

`sandhya.ravi@GT9X7CVF5F kommander-applications % trivy image ghcr.io/mesosphere/dkp-container-images/docker.io/bitnami/postgresql:15.2.0-debian-11-r21-d2iq.0  --vuln-type os --ignore-unfixed | grep Total

2024-07-12T13:32:24+05:30	INFO	Vulnerability scanning is enabled
2024-07-12T13:32:24+05:30	INFO	Secret scanning is enabled
2024-07-12T13:32:24+05:30	INFO	If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2024-07-12T13:32:24+05:30	INFO	Please see also https://aquasecurity.github.io/trivy/v0.53/docs/scanner/secret#recommendation for faster secret detection
2024-07-12T13:32:27+05:30	INFO	Detected OS	family="debian" version="11.6"
2024-07-12T13:32:27+05:30	INFO	[debian] Detecting vulnerabilities...	os_version="11" pkg_num=113
2024-07-12T13:32:27+05:30	WARN	Using severities from other vendors for some vulnerabilities. Read https://aquasecurity.github.io/trivy/v0.53/docs/scanner/vulnerability#severity-selection for details.
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)`

workflow link:

**Which issue(s) does this PR fix?**:

https://jira.nutanix.com/browse/NCN-101394
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

**Comment by SandhyaRavi2403** [comment]
@mhrabovcin , even after updating the images in the cm.yaml, getting the below error. am I missing any reference here.
`Error: No mapping for image "docker.io/bitnami/postgresql:15.2.0-debian-11-r21"`

File: services/git-operator/0.1.0/git-operator-manifests/all.yaml [hunk 1]
```diff
@@ -699,7 +699,7 @@ spec:
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
         - --namespace=${NAMESPACE:=git-operator-system}
-        image: docker.io/mesosphere/git-operator:v0.8.2@sha256:7dde5a0836d212d8fe31f65c42bec32c91962858f6cb7da5526a0b60727141fb
+        image: docker.io/mesosphere/git-operator:v0.8.3@sha256:3a4c42890549394bbe3842d99c7bf143334c19dc6e8d13413cd796c219aead8e
```
Comment (mhrabovcin) [inline services/git-operator/0.1.0/git-operator-manifests/all.yaml:702]: Can you rebase your PR as this change seems unrelated.

Comment (SandhyaRavi2403) [reply services/git-operator/0.1.0/git-operator-manifests/all.yaml]: done

File: services/gitea/8.2.1/defaults/cm.yaml
Comment (mhrabovcin) [inline services/gitea/8.2.1/defaults/cm.yaml:1]: This needs to rebased, the `gitea` was removed from this repo.

Comment (mhrabovcin) [inline services/gitea/8.2.1/defaults/cm.yaml:1]: Gitea was removed from this repo.

File: .github/service-labeler.yaml [hunk 2]
```diff
@@ -54,6 +54,10 @@ services/git-operator:
 - changed-files:
   - any-glob-to-any-file:
     - services/git-operator/**
+services/gitea:
+- changed-files:
+  - any-glob-to-any-file:
+    - services/gitea/**
```
Comment (mhrabovcin) [inline .github/service-labeler.yaml:60]:
```suggestion
```

Gitea was removed from this repo.

File: licenses.d2iq.yaml [hunk 3]
```diff
@@ -4,8 +4,8 @@ ignore:
   - docker.io/nginxinc/nginx-unprivileged:1.24.0-alpine
   - docker.io/nginxinc/nginx-unprivileged:1.25.4-alpine
   - docker.io/bitnami/external-dns:0.14.2-debian-12-r1
-  - docker.io/bitnami/postgresql:11.22.0-debian-11-r4
-  - docker.io/bitnami/postgresql:15.2.0-debian-11-r21
+  - ghcr.io/mesosphere/dkp-container-images/docker.io/bitnami/postgresql:11.22.0-debian-11-r4-d2iq.0
```
Comment (mhrabovcin) [inline licenses.d2iq.yaml:7]: This needs to be removed.

**Review by asekretenko — CHANGES_REQUESTED** [review]
Any changes into dkp-insights `cm.yaml` should land into the `dkp-insights` repository, not here.
https://github.com/mesosphere/dkp-insights/blob/main/charts/dkp-insights/values.yaml

The default values configmap here is autogenerated (and not used on the daily cluster where Insights are installed via Helm command line).

**Comment by SandhyaRavi2403** [comment]
> Any changes into dkp-insights `cm.yaml` should land into the `dkp-insights` repository, not here. https://github.com/mesosphere/dkp-insights/blob/main/charts/dkp-insights/values.yaml
> 
> The default values configmap here is autogenerated (and not used on the daily cluster where Insights are installed via Helm command line).

@asekretenko, @mhrabovcin 
As mentioned raised a PR in dkp-insights, https://github.com/mesosphere/dkp-insights/pull/1515

How do we test it ? Is it that once the dkp PR is merged the image details are updated in default cm.yaml of kapps ?

**Comment by asekretenko** [comment]
@SandhyaRavi2403 Normally we do a new release of Insights and bump the app in `kommander-applications` right before cutting a DKP release/candidate. The daily cluster machinery installs Insights by using helm command line directly; the Insights app is not involved in that.

That said, there is no need to do anything in `kommander-applications` as part of the CVE fix in Insights.

From what I hear from @alembiewski, similar logic applies to the AP Navigator: the image should be bumped in the Helm chart.

File: services/ai-navigator-app/0.2.3-d2iq.0/defaults/cm.yaml [hunk 4]
```diff
@@ -56,7 +56,7 @@ data:
         priorityClassName: dkp-high-priority
       enabled: true
       image:
-        tag: 15.2.0-debian-11-r21
+        tag: 15.2.0-debian-11-r21-d2iq.0
```
Comment (SandhyaRavi2403) [inline services/ai-navigator-app/0.2.3-d2iq.0/defaults/cm.yaml:59]: @asekretenko, do we have to retain the same image tag or feasible to change to updated version ?

**Comment by SandhyaRavi2403** [comment]
closing this PR, as the changes are updated in https://github.com/mesosphere/kommander-applications/pull/2459
